### DISCUSSION TITLE
Algum outro framework além do Bootstrap?

### DISCUSSION THREAD

**Discussion by patife1** [community/community#146041] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Estou utilizando o Bootstrap, porém, gostaria de saber se existe algum outro framework que eu possa utilizar em meu site que fosse tão prático como o Bootstrap.

**Comment by marquescharlon** [comment, marked as answer]
Olá, bom dia!

O Zurb Foundation é uma boa alternativa.
Você pode encontrá-lo através do link https://get.foundation/

É bem semelhante ao Boostrap, precisa adicionar as dependências em seu projeto, então, utilizar os componentes encontrados em https://get.foundation/sites/docs/
